code/ui/blocks/src/examples/ControlsParameters.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { ControlsParameters } from './ControlsParameters';
+
+/**
+ * Reference stories to be used by the Controls stories
+ */
+const meta = {
+  title: 'Example/Stories for Controls',
```

Comment:
we have a whole section called Controls, so we might want to make this more explicit.
```suggestion
  title: 'Example/Stories for the Controls Block',
```